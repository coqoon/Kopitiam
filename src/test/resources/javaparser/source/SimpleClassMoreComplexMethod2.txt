class Foo { int foo (int a) { return foo(foo(a)); } }