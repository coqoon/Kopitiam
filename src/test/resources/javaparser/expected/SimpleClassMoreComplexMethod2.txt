class Foo {
  int foo (int a) {
    int tmp_1 = this.foo(a);
    int tmp_2 = this.foo(tmp_1);
    return tmp_2;
  }
}